source/_cookbook/dim_and_brighten_lights.markdown

```diff
@@ -0,0 +1,164 @@
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 13
+          entity_id: zwave.YOURCONTROLLER
+    action:
+      - service: script.turn_on
+        data:
+          entity_id: script.light_bright
+
+  - alias: 'Stop the bright just there'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 15
+          entity_id: zwave.YOURCONTROLLER
+    action:
+      - service: script.turn_off
+        data_template:
+          entity_id: script.light_bright
+      - service: script.turn_off
+        data:
+          entity_id: script.light_bright_pause
+
+  - alias: 'Make the lights go dim'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 14
+          entity_id: zwave.YOURCONTROLLER
+    action:
+      - service: script.turn_on
+        data:
+          entity_id: script.light_dim
+
+  - alias: 'Stop the dim just there'
+    initial_state: 'on'
+    trigger:
+      - platform: event
+        event_type: zwave.scene_activated
+        event_data:
+          scene_id: 16
+          entity_id: zwave.YOURCONTROLLER
+    action:
+      - service: script.turn_off
+        data_template:
+          entity_id: script.light_dim
+      - service: script.turn_off
+        data:
+          entity_id: script.light_dim_pause
+```
+
+There are 2 variables that control the speed of the change for the scripts below. The first is the `step` - small steps create a smooth transition. The second is the delay, larger delays will create a slower transition.
+
+To allow flexibility, we'll use an input_slider for the step (at the time of writing this, it's not possible to template the delay when the delay uses milliseconds). We're also using a slider to set the minimum and maximum brightness, so that it's easy to tune that (or manage it through an automation).
+
+```yaml
+input_slider:
+  light_step:
+    name: 'Step the lights this much'
+    initial: 20
+    min: 1
+    max: 64
+    step: 1
+
+  light_minimum:
+    name: 'No dimmer than this'
+    initial: 5
+    min: 1
+    max: 255
+    step: 1
+    
+  light_maximum:
+    name: 'No brighter than this'
+    initial: 255
+    min: 50
+    max: 255
+    step: 1
+```
+
+Now the scripts. There are 2 pairs of scripts. The first steps the light brighter to the maximum, and the second provides the delay. These call each other until both are stopped. The second pair do the same for dimming.
+
+```yaml
+# Replace YOURLIGHT with the actual light entity
+script:
+    light_bright:
+      sequence:
+        - service: light.turn_on
+          data_template:
+            entity_id: light.YOURLIGHT
+            brightness: >-
+              {% raw %}{% set current = (states.light.YOURLIGHT.attributes|default).brightness|default(0)|int %}
+              {% set step = states.input_slider.light_step.states|int %}
```

Comment:
`states.input_slider.light_step.states|int` -> `states('input_slider.light_step')|int`

There are 5 more lines similar to this that need to be changed as well.